source/features/reactions-avatars.tsx

```diff
@@ -61,8 +61,8 @@ async function showAvatarsOn(commentReactions: Element): Promise<void> {
 	for (const {container, username, imageUrl} of flatParticipants) {
 		container.append(
 			// Without this, Firefox will follow the link instead of submitting the reaction button
-			<a href={isFirefox ? undefined : `/${username}`}>
-				<img src={imageUrl}/>
+			<a href={isFirefox ? undefined : `/${username}`} className="rounded-1 avatar-user">
+				<img src={imageUrl} className="avatar-user rounded-1"/>
```

Comment:
Why do we need `className` on child too?

Reply:
Class only on `<a>`:
![UPS89IrGl9](https://user-images.githubusercontent.com/44045911/84990640-049fe100-b178-11ea-917c-edfc94d4d6c8.gif)
Class only on `<img>`:
![e3PtlT0qQg](https://user-images.githubusercontent.com/44045911/84990684-1aada180-b178-11ea-9a07-bd703c05dd4e.gif)

Reply:
> You can see there was `border-radius: inherit` for `.reaction-summary-item a img`.

Yes, but that is removed in this PR. Instead you can `inherit` in and add classes just to the anchor tag.

This way, we can retain compatibility for users who are not on preview.

Class only on `<a>` with `border-radius: inherit` left in:


![image](https://user-images.githubusercontent.com/37769974/84996742-4d04d200-b16b-11ea-882c-2f8302f3c6b7.png)

Reply:
> Instead you can `inherit` in and add classes just to the anchor tag.

I see.